src/main/java/edu/cornell/kfs/module/purap/CUPurapConstants.java

```diff
@@ -135,6 +146,12 @@ public static final class IWantPresentationModes {
     public static final String IWNT_DOC_USE_LOOKUPS = "iwntUseLookups";
     public static final String IWNT_DOC_DISPLAY_NOTE_OPTIONS = "displayNoteOptions";
     
+    public static final String IWNT_DOC_DISPLAY_CONTRACT_TAB = "displayContractTab";
+    public static final String IWNT_DOC_EDIT_CONTRACT_INDICATOR = "editContractIndicator";
+    public static final String IWNT_DOC_DISPLAY_CONFIRMATION = "displayConfirmation";

```

Comment:
On a second thought I wonder if we should leave this in place and have a separate user story to investigate if support for java-script being turned off should be removed. Please let me know what you think. Thank you.

Reply:
As an observer to this conversation, I would vote for a separate user story for that java-script being turned off investigation. It would allow for a more focused and scoped unit of work.

Reply:
Yes, I agree that it makes sense to create a separate user story for potentially removing the no-JavaScript handling. I'm fine with the current dual JavaScript and no-JavaScript handling while we await that future analysis.